spec/lib/miq_expression_spec.rb

```diff
@@ -2983,4 +2991,8 @@
       )
     end
   end
+
+  def stub_user_with_timezone(timezone)
```

Comment:
I think we already have a stub user concept in auth helper.
Also, I'm a little concerned with stubbing the time zone in general here.

Would be nice if the user class had a timezone and we didn't have to stub it.